crates/turborepo-lib/src/run/watch.rs

```diff
@@ -120,9 +120,14 @@ impl WatchClient {
         let signal = get_signal()?;
         let handler = SignalHandler::new(signal);
 
-        // Check if the turbo.json path is the standard one
-        let standard_path = base.repo_root.join_component(CONFIG_FILE);
-        if base.opts.repo_opts.root_turbo_json_path != standard_path {
+        // Check if the turbo.json path is the standard one (either turbo.json or
+        // turbo.jsonc)
+        let standard_path_json = base.repo_root.join_component(CONFIG_FILE);
+        let standard_path_jsonc = base.repo_root.join_component(CONFIG_FILE_JSONC);
+
+        if base.opts.repo_opts.root_turbo_json_path != standard_path_json
+            && base.opts.repo_opts.root_turbo_json_path != standard_path_jsonc
```

Comment:
I haven't been in any of the code for `watch` and have little context so not sure if this is an okay way to go about it. I just did the first, most minimally naive thing I saw.